widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java

```diff
@@ -43,6 +43,7 @@ public class CallActivity extends AppCompatActivity {
         navigateToChat();
         finish();
     };
+    private final CallView.OnTitleUpdatedListener onTitleUpdatedListener = this::setTitle;
```

Comment:
no reason to keep this callback global, you can just inline it. `callView.setOnTitleUpdatedListener(this::setTitle);`

Reply:
done